neonvm/controllers/virtualmachine_controller.go

```diff
@@ -955,6 +989,43 @@ func (r *VirtualMachineReconciler) podForVirtualMachine(
 	return pod, nil
 }
 
+func (r *VirtualMachineReconciler) sshSecretForVirtualMachine(virtualmachine *vmv1.VirtualMachine) (*corev1.Secret, error) {
```

Comment:
You should probably use same formatting as for the `podForVirtualMachine` function

Reply:
I tried to do the same, but I wanted to be a little bit more explicit so I named it `sshSecretForVirtualMachine` rather than `secretForVirtualMachine`.

Reply:
I meant the line too long and arguments should be on new line each. But now I see it's only one argument :fearful: 

Also, I think argument should be named `vm` and not `virtualmachine`.  "Variable names in Go should be short rather than long" after all [1]. 

I know that's not consistent throughout this file, but we should make it shorter everywhere.  

1: [Variable names in Go should be short rather than long](https://go.dev/wiki/CodeReviewComments#variable-names)

Reply:
I agree but I want to keep it consistent with the rest of code in the controller.